common/text/text_structure_test.cc

```diff
@@ -476,7 +475,7 @@ TEST_F(TextStructureViewPublicTest, ExpandSubtreesOneLeaf) {
   const int divide = 3;
   const int new_node_tag = 7;
   std::string subtext(tokens_[0].text().data(), tokens_[0].text().length());
-  auto subanalysis = absl::make_unique<TextStructure>(subtext);
+  std::unique_ptr<TextStructure> subanalysis(new TextStructure(subtext));
```

Comment:
same thought about `operator new` from earlier comment, but I don't mind the type-repetition much.

Reply:
Same reason as above why I had to change this away from make_unique.